anthos-bm-utils/abm-nettest/README.md

```diff
@@ -0,0 +1,105 @@
+## What is this?
+
+ABM nettest enables users/support members to find any connectivity issues
+among k8s pods/nodes/svcs/external targets (except for connections from external
+targets to pods/nodes/svcs). This playbook will help to quickly spot any
+connectivity issues among k8s components in the cluster of interest.
+
+## Quickstart
+
+### Run Nettest
+
+Run the nettest by running the following commands.
+
+For non-RHEL OS:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/anthos-samples/main/anthos-bm-utils/abm-nettest/nettest.yaml
+```
+
+For RHEL OS:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/anthos-samples/main/anthos-bm-utils/abm-nettest/nettest_rhel.yaml
+```
+
+NOTE: private registry users needs to download the manifest and modify all image names respectively.
+
+### Get test results
+
+Run the following command to see the nettest result:
```

Comment:
Do I run this immediately after deployment, or anytime I want to see results? This is just porting the logs to stdout, right? Does run one time and stop? If `nettest` finds an issue. How do I re-run the test after I've resolved the issue?

Reply:
I added a line that it will start automatically in the above section `Run nettest`